hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemRandomRead.java

```diff
@@ -402,6 +399,18 @@ public void testSkipAndAvailableAndPosition() throws Exception {
               inputStream.getPos());
       assertEquals(testFileLength - inputStream.getPos(),
               inputStream.available());
+
+      skipped = inputStream.skip(testFileLength + 1); //goes to last byte
+      assertEquals(1, inputStream.available());
```

Comment:
can you add an error message to all these asserts?

Reply:
added